bool get_dock
---
string first_dock_symbol
string first_dock_color
string second_dock_symbol
string second_dock_color